config/environments/development.rb

```diff
@@ -3,7 +3,28 @@
 
    config.action_mailer.default_url_options = { host: 'localhost:3000' }
 
-
+   config.action_mailer.delivery_method = :smtp
+   
+  # THIS IS SENDGRID
+  # config.action_mailer.smtp_settings = {
+  #   :user_name => 'kristen@girldevelopit.com',
+  #   :password => 'gc3CfScU)^?,64p',
+  #   :domain => 'girl-develop-it.herokuapp.com',
+  #   :address => 'smtp.sendgrid.net',
+  #   :port => 587,
+  #   :authentication => :plain,
+  #   :enable_starttls_auto => true
+  # }
+
```

Comment:
Deleted Sendgrid and trust your comparison testing! :raised_hands: Please remove lines 8-17 to avoid future confusions.